geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementService.java

```diff
@@ -47,6 +48,7 @@
    *        cluster
    * @return a {@link ClusterManagementRealizationResult} indicating the success of the deletion
    * @throws IllegalArgumentException, NoMemberException, EntityExistsException
```

Comment:
Having the NoMemberException and EntityExistsException in the external api is a problem because they are internal. So users could not get code to compile that wants to handle these and the javadocs will have broken links. We could just have one external exception class that these apis throw when they fail. Internally we could catch the other exceptions and throw the external one.

Reply:
replaced with our own exception